03-reproducibility.Rmd

```diff
@@ -31,10 +31,6 @@ Recommended readings on the "reproducibility crisis":
 
 Appropriate study design is beyond the scope of this lab manual and is something trainees develop through their coursework and mentoring. 
 
-## Register study protocols
-
-We register all randomized trials on [clinicaltrials.gov](clinicaltrials.gov), and in some cases register observational studies as well. 
-
 ## Write and register pre-analysis plans
```

Comment:
I am intrigued by the idea of a pre-analysis plan but I haven't used them in my previous workflow yet. Maybe remove this section for now? we can add it back later when it becomes a regular practice for projects in our lab.